app/commands/webhooks/process_push_update.rb

```diff
@@ -15,12 +15,23 @@ def call
       else
         track = Track.find_by(slug: repo_name)
         SyncTrackJob.perform_later(track) if track
+        NotifyTrackOrgWideFileChangeJob.perform_later(track) if org_wide_file_changed?(track)
       end
     end
 
     private
     def pushed_to_main?
       ref == "refs/heads/#{Git::Repository::MAIN_BRANCH_REF}"
     end
+
+    def org_wide_file_changed?(track)
+      return false unless track
+
+      commits.to_a.any? do |commit|
+        commit[:added].concat(commit[:removed], commit[:modified]).any? do |file|
```

Comment:
`concat`. Posh.

Reply:
Oh, that's just my functional background :D

Reply:
It's nice. As purely an FYI, I'd have done `Set.new([*commit[:added], *commit[:removed], *commit[:modified]])` probably.